.github/workflows/base.yml

```diff
@@ -0,0 +1,30 @@
+name: Base image build
+
+on: 
+  workflow_dispatch:
+    branches: 
+     - '*'
+     - '!flatcap'
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1  
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1 
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner  }}
+        password: ${{ secrets.GH_TOKEN }}
+    - name: Build and push
+      uses: docker/build-push-action@v2
+      with:
+        file: base.Dockerfile
+        platforms: linux/amd64
+        push: true
+        tags: |
+            ghcr.io/nixmark/neomutt-base:latest
```

Comment:
Shouldn't the namespace be `neomutt` ?

Reply:
Yes, this is my bad. I tested everything in my repo, and forgot to fix this. I will fix in next commit.